upgrade/1.5.2/README.md

```diff
@@ -281,9 +281,9 @@ This step does not rebuild NCNs. These new images are built and stored in S3 to
 
 1. Choose one of the following options based on the output from the previous step.
 
-    * Option 1: [Upgrade of CSM on system with additional products](../Stage_0_Prerequisites.md#option-1-upgrade-of-csm-and-additional-products)
-    * Option 2: [Upgrade of CSM on CSM-only system](./CSM-Only.md#steps) ***(Do not use this procedure if more than CSM is
-      installed on the system.\)***
+    * Option 1: [Upgrade of CSM on system with additional products](./CSM-With-Other-Products.md)
+    * Option 2: [Upgrade of CSM on CSM-only system](./CSM-Only.md#steps) _(Do not use this procedure if more than CSM is
+      installed on the system.\)_
```

Comment:
What is the correct formatting for these underscores? Checks say they are misspelled

Reply:
I'm not sure. I don't see underscores as a normal markdown symbol on [this cheat sheet](https://www.markdownguide.org/cheat-sheet/)

Maybe replace them with asterisks or double asterisks, if italicizing or bolding is acceptable

Reply:
Yeah, moving the _-surrounded text to the same line fixed the lint complaint